paasta_tools/deploy_marathon_services

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
 if am_i_mesos_leader >/dev/null; then
-  list_marathon_service_instances | shuf | xargs -n 1 -r -P 5 setup_marathon_job
+  list_marathon_service_instances | xargs -n 64 -r -P 4 setup_marathon_job
```

Comment:
Also, for best practice can you use the long-form of these xargs args to make it so I don't have to remember or lookup what they mean?